sphinxcontrib/drawio/__init__.py

```diff
@@ -240,8 +248,8 @@ def _drawio_export(self, input_abspath, options, out_filename):
             "--format",
             output_format,
             "--output",
-            str(export_abspath),
-            str(input_abspath),
+            str(export_abspath_cygwin if export_abspath_cygwin is not None else export_abspath),
```

Comment:
Since `export_abspath_cygwin` overrides `export_abspath` anyway, why not just override `export_abspath` on lines 216-219?